### ISSUE TITLE
Windows: exits after entering master password

### ISSUE THREAD

**Issue by bourbonspecial** [renatoathaydes/go-hash#2]
Running go-hash command on windows 10, the tool exits immediately after entering the master password so you can't use any sub-commands.

**Comment by bourbonspecial** [comment]
Running under wsl (windows subsystem for linux) is a bit of a sledgehammer but gets the job done!

**Comment by renatoathaydes** [comment]
Do you get any errors on cmd.exe?

**Comment by bourbonspecial** [comment]
No. I get prompted for the master password then it exits and goes back to the cmd.exe command prompt.

**Comment by renatoathaydes** [comment]
Hm. The problem is probably with the readline library... Can you try to run this demo?

https://github.com/chzyer/readline/blob/master/example/readline-demo/readline-demo.go

If that works, then the next suspect is the password reader: golang.org/x/crypto/ssh/terminal

But that seems to be working...

**Comment by renatoathaydes** [comment]
Hi!

I fixed some issues on Windows 7, but I cannot guarantee it works in Windows 10 because I have lots of problems on Windows 7 related to readline.

See this bug:

https://github.com/chzyer/readline/issues/21

Are you able to pull the master branch, run "go install" and see if that works?

**Comment by renatoathaydes** [comment]
@bourbonspecial let me know if you can test the master branch (by installing with `go get -u github.com/renatoathaydes/go-hash`) or if you need a binary.

**Comment by renatoathaydes** [comment]
Released version 0.7.0 which should print any errors that occur... if you see an error, I 'm afraid you can't use go-hash in your Windows version. Please use the Linux sub-system instead.

**Comment by sergeevabc** [comment]
```
$ go-hash-windows-amd64.exe test

Go-Hash version GH01

No database exists yet, to create one, you need to provide a strong password first.
A strong password could be a phrase you could remember easily but that is hard to guess.
To make it harder to guess, include both upper and lower-case letters, numbers and special characters like ? and @.
If you forget this password, there's no way to recover it or your data, so be careful!

Please enter a master password:
Re-enter the password:

Welcome, go-hash at your service.

go-hash» help

go-hash commands:

  cp       copies an entry's field to the clipboard. Fields: -p = password, -u = username.
  goto     goes to the URL associated with an entry and copies its password to the clipboard.
  cmp      changes the master password.
  help     prints this message or help about a specific command.
  group    manages/enters groups.
  entry    manages entries within the current group.

Type 'exit' to exit a group or quit if you are not within a group.
To quit from anywhere, type 'quit'.

go-hash» fatal error: all goroutines are asleep - deadlock!

goroutine 1 [select]:
github.com/renatoathaydes/go-hash/vendor/github.com/chzyer/readline.(*Operation).Runes(0xc04209a180, 0x0, 0x0, 0x0, 0x0, 0x0)
        /home/travis/gopath/src/github.com/renatoathaydes/go-hash/vendor/github.com/chzyer/readline/operation.go:350 +0x1fc
github.com/renatoathaydes/go-hash/vendor/github.com/chzyer/readline.(*Operation).String(0xc04209a180, 0xc, 0x12, 0x14, 0xc042076370)
        /home/travis/gopath/src/github.com/renatoathaydes/go-hash/vendor/github.com/chzyer/readline/operation.go:337 +0x32
github.com/renatoathaydes/go-hash/vendor/github.com/chzyer/readline.(*Instance).Readline(0xc042044e40, 0xc04204c3a0, 0x13, 0xc042076370, 0x12)

        /home/travis/gopath/src/github.com/renatoathaydes/go-hash/vendor/github.com/chzyer/readline/readline.go:216 +0x36
main.runCliLoop(0xc04206e4e8, 0xc04204e088, 0x4, 0xc04204e430, 0x8)
        /home/travis/gopath/src/github.com/renatoathaydes/go-hash/main.go:141 +0x3f9
main.main()
        /home/travis/gopath/src/github.com/renatoathaydes/go-hash/main.go:237 +0x28e
```
